### ISSUE TITLE
Inactive Repo

### ISSUE THREAD

**Issue by mallorysmith64** [WWF-International/animal-iq#3]
@noahpatterson @hfinn 
Hi, 

This repo has not had a commit in 7 years. I would really like to make some contributions. Is there a way I could get at least some **deployment rights** or perhaps even **write access** to the repository and take over?

Thanks.

**Comment by noahpatterson** [comment]
@mallorysmith64 we've been maintaining a more sharable version of this in a different repo on Gitlab. Could you share with me your gitlab username and I can give you access? Then, you can let me know the changes you're looking to make.

**Comment by mallorysmith64** [comment]
@noahpatterson Oh, okay. I didn't know that. My GitLab username is mallorysmith64.

**Comment by noahpatterson** [comment]
Hi Mallory,  happy to give you access, first though I’d like to verify
which office you are working with? Is it WWF-int? Who is your contact or
direct report there?

Thanks

**Comment by mallorysmith64** [comment]
@noahpatterson Hi, Noah. I do not work at an office at WWF. I'm a volunteer developer. I'm wanting to add some more animal quizzes to this project. Does the "animal-iq" repo on here have a different name on GitLab? I am searching for it, but I am unable to find it.

Thanks for your help.

**Comment by noahpatterson** [comment]
Ah ok! Well thanks for wanting to help! The git lab repo is not public. Let
me review and make sure there isn’t any organizational secret information
in the repo.p and perhaps I can make it public.

I’ve had some ideas about updating this projects code to be more modern. If
you were interested in working on a code overhaul, perhaps we can setup
some time to chat?

**Comment by mallorysmith64** [comment]
@noahpatterson Yeah, I would be interested in setting up a time to chat! 😀 What's the best way to contact you, so I can message you more privately about this? I usually use Google Hangouts, but we use whatever you want.

Thanks.
